src/containers/CreateBounty/index.js

```diff
@@ -64,23 +71,145 @@ class CreateBountyComponent extends React.Component {
       );
     }
 
+    var prompter = undefined;
+    if (
+      this.state.dirty &&
+      !isEditing &&
+      (this.state.dirty && !isDraft) &&
+      (this.state.dirty && this.state.submitNotPressed)
+    ) {
+      prompter = (
+        <NavigationPrompt
+          renderIfNotActive={false}
+          when={(crntLocation, nextLocation) =>
+            !nextLocation ||
+            !nextLocation.pathname.startsWith(crntLocation.pathname)
+          }
+        >
+          {({ isActive, onCancel, onConfirm }) => {
+            if (isActive) {
+              return (
+                <Modal
+                  dismissable
+                  size={'medium'}
+                  fixed
+                  visible={true}
+                  onClose={onCancel}
+                >
+                  <Modal.Header closable />
+                  <Modal.Body>
```

Comment:
For our modal component, formatting is built in, and you shouldn't need to add any additional classNames or styles. For reference, take a look at `src/containers/Bounty/components/modals/ContributeFormModal.js` to see how `Modal.Message` and `Modal.Description` are used to determine how different elements of the modal are structured and styled. Please remove any inline styles and HTML tags and defer to composing the modal in this way.

As others has also mentioned, please replace the text content with copy added to `src/i18n/en_US.json`.